docs/developers_guide/ocean/api.md

```diff
@@ -99,6 +99,27 @@
    viz.Viz.run
 ```
 
+### single_column
+
+```{eval-rst}
+.. currentmodule:: polaris.ocean.tests.single_column
+
+.. autosummary::
+   :toctree: generated/
+
+   SingleColumn
+
+   forward.Forward
+
+   initial_state.InitialState
+   initial_state.InitialState.run
+
+   viz.Viz
+   viz.Viz.run
+
+   cvmix.CVMix
+   cvmix.CVMix.validate
```

Comment:
````suggestion
   cvmix.CVMix
   cvmix.CVMix.validate
```
````

Reply:
I'm getting a bunch of errors when I try to build the docs locally because of this (it gets very confused about `OceanModelStep` below).